naresh/src/main/java/com/lftechnology/phpjava/assignmenttwo/MutiplierChecker.java

```diff
@@ -0,0 +1,56 @@
+package com.lftechnology.phpjava.assignmenttwo;
+
+import com.lftechnology.phpjava.menu.MenuReader;
+import com.lftechnology.phpjava.menu.ProgramMenu;
+
+import java.util.ArrayList;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.ArrayList'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)